src/NuGet.Services.EndToEnd/AutoCompleteResultTests.cs

```diff
@@ -103,5 +103,44 @@ public async Task V2PackageIdsAutocompleteResultsMatchV3Results(
                 Assert.Equal(v2Response, v3Response.Data);
             }
         }
+
+        [GalleryTestTheory]
+        [InlineData(PackageType.SemVer1Stable, true, null)]
```

Comment:
I think you'd want a new package type here, like `PackageType.UnicodeId` or something.

Reply:
I pushed another file